pkg/storaged/storaged.jsx

```diff
@@ -49,7 +49,12 @@ class Application extends React.Component {
     componentDidMount() {
         client.addEventListener("changed", this.on_client_changed);
         cockpit.addEventListener("locationchanged", this.on_navigate);
-        client.init(() => { this.setState({ inited: true }) });
+        client.init(() => {
+            if (client.in_anaconda_mode()) {
+                document.body.classList.add("anaconda");
```

Comment:
Shouldn't we keep the changes to `pages.jsx` from the previous version of this PR?  The ones 

```
- <Page id="storage" className={"no-masthead-sidebar" + (client.in_anaconda_mode() ? " anaconda" : "")}>
+ <Page id="storage" className="no-masthead-sidebar">
```